perfkitbenchmarker/linux_benchmarks/cloudsuite_web_search_benchmark.py

```diff
@@ -0,0 +1,195 @@
+# Copyright 2015 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Runs the Web Search benchmark of Cloudsuite.
+
+More info: http://cloudsuite.ch/websearch/
+"""
+
+import re
+import time
+
+from perfkitbenchmarker import configs
+from perfkitbenchmarker import flags
+from perfkitbenchmarker import sample
+from perfkitbenchmarker import vm_util
+from perfkitbenchmarker.linux_packages import docker
+
+FLAGS = flags.FLAGS
+
+flags.DEFINE_string('cloudsuite_web_search_client_heap_size',
+                    '2g',
+                    'Java heap size for Faban client in the usual java format.'
+                    ' Default: 2g.')
+
+flags.DEFINE_string('cloudsuite_web_search_server_heap_size',
+                    '3g',
+                    'Java heap size for Solr server in the usual java format.'
+                    ' Default: 3g.')
+flags.DEFINE_enum('cloudsuite_web_search_query_distr',
+                  'Random',
+                  ['Random', 'Ziphian'],
+                  'Distribution of query terms. '
+                  'Random and Ziphian distributions are available. '
+                  'Default: Random.')
+flags.DEFINE_integer('cloudsuite_web_search_num_clients',
+                     1,
+                     'Number of client machines.',
+                     lower_bound=1)
+flags.DEFINE_integer('cloudsuite_web_search_ramp_up',
+                     90,
+                     'Benchmark ramp up time in seconds.',
+                     lower_bound=1)
+flags.DEFINE_integer('cloudsuite_web_search_ramp_down',
+                     60,
+                     'Benchmark ramp down time in seconds.',
+                     lower_bound=1)
+flags.DEFINE_integer('cloudsuite_web_search_steady_state',
+                     60,
+                     'Benchmark steady state time in seconds.',
+                     lower_bound=1)
+flags.DEFINE_integer('cloudsuite_web_search_scale',
+                     50,
+                     'Number of simulated web search users.',
+                     lower_bound=1)
+
+BENCHMARK_NAME = 'cloudsuite_web_search'
+BENCHMARK_CONFIG = """
+cloudsuite_web_search:
+  description: >
+    Run Cloudsuite Web Search benchmark. Specify the number of
+    clients with --num_vms.
```

Comment:
This description says `num_vms`, which matches the `GetConfig` behavior, but then what is `cloudsuite_web_search_num_clients` used for?